### PR TITLE
librealsense2: 2.54.1-2 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
Increasing version of package(s) in repository `librealsense2` to `2.54.1-2`:

- upstream repository: https://github.com/IntelRealSense/librealsense.git
- release repository: https://github.com/IntelRealSense/librealsense2-release.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

### CODE REVIEW

File: rolling/distribution.yaml
Comment (clalancette) [inline rolling/distribution.yaml:2346]: To release into Rolling, we need to have the release repository be in the https://github.com/ros2-gbp organization.  The basic reason is so that we have write access when we go to branch off of Rolling.  To create that, please open a ticket on https://github.com/ros2-gbp/ros2-gbp-github-org

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rolling/distribution.yaml
Comment (SamerKhshiboun) [reply rolling/distribution.yaml:2346]: Opened PR, waiting for merge:
https://github.com/ros2-gbp/ros2-gbp-github-org/pull/297

Comment (Nir-Az) [reply rolling/distribution.yaml:2346]: @clalancette Hi,
We opened a PR 2 weeks ago as you asked.
Can this please be reviewed / merged?

**Comment by sloretz** [comment]
Holding for an [upcoming Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2023-08-09/32782)

**Comment by sloretz** [comment]
Holding for an upcoming [Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2023-09-27/33770)

**Comment by clalancette** [comment]
I'm going to close this PR until the discussion in https://github.com/ros2-gbp/ros2-gbp-github-org/pull/297 is sorted out.  Once that is done, a new PR is going to have to be opened anyway.

**Pull request closed without merging**
